assets/js/googlesitekit/modules/datastore/modules.js

```diff
@@ -144,6 +144,27 @@ const fetchSetModuleActivationStore = createFetchStore( {
 	},
 } );
 
+const fetchCheckModuleAccessStore = createFetchStore( {
+	baseName: 'checkModuleAccess',
+	controlCallback: ( { slug } ) => {
+		return API.set( 'core', 'modules', 'check-access', { slug } );
+	},
+	reducerCallback: ( state, { access }, { slug } ) => {
+		return {
+			...state,
+			moduleAccess: {
```

Comment:
We also need to add the `moduleAccess` property to the `baseInitialState` object with `{}` as the default value.